integration/scripts/deploy.sh

```diff
@@ -6,9 +6,9 @@
 # environments:
 #
 # * Developer emulated xPU setup
-# * Split setup between an actual xPU and a host/VM/container
+# * Split setup between an actual xPU and a laptop/VM/server
```

Comment:
Could be container as well, in fact, I was testing that out locally. But fine for now.